src/main/java/frc/robot/SwerveModule.java

```diff
@@ -97,11 +97,6 @@ public double makePositiveDegrees(Rotation2d anAngle){
         return makePositiveDegrees(anAngle.getDegrees());
     }
 
-    public Rotation2d getAngle(){
```

Comment:
Why get rid of getAngle()?

Reply:
It uses a deprecated method, and I was trying to figure out how to replace it when I realized that it is never used anywhere in our code ever. If we need to add it back we can do so later.